src/collection/collectionmodel.h

```diff
@@ -173,7 +176,7 @@ class CollectionModel : public SimpleTreeModel<CollectionItem> {
   static QString PrettyDisc(const int disc);
   static QString SortText(QString text);
   static QString SortTextForNumber(const int number);
-  static QString SortTextForArtist(QString artist);
+  static QString SortTextForArtist(QString artist, bool skip_articles);
```

Comment:
const bool skip_articles

Reply:
Done, but note that in declarations (as opposed to definitions), it makes no difference:

https://abseil.io/tips/109

> Never use top-level `const` on function parameters in _declarations_ that are not definitions (and be careful not to copy/paste a meaningless `const`). It is meaningless and ignored by the compiler, it is visual noise, and it could mislead readers.